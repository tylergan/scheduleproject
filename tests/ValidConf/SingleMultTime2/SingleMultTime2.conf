at 1200,1300 run /bin/echo Hello