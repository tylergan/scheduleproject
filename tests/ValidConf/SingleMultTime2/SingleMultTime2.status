will run at XX XX XX 12:00:00 2020 /bin/echo Hello
will run at XX XX XX 13:00:00 2020 /bin/echo Hello
ran XX XX XX 12:00:00 2020 /bin/echo Hello
will run at XX XX XX 13:00:00 2020 /bin/echo Hello
ran XX XX XX 12:00:00 2020 /bin/echo Hello
ran XX XX XX 13:00:00 2020 /bin/echo Hello
