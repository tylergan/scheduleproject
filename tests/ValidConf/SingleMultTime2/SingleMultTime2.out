Hello
Hello
nothing left to run
